src/NUnitFramework/framework/Attributes/TestCaseSourceAttribute.cs

```diff
@@ -216,7 +216,7 @@ private IEnumerable GetTestCaseSource(IMethodInfo method)
                 return Reflect.Construct(sourceType) as IEnumerable;
 
             MemberInfo[] members = sourceType.GetMember(SourceName,
-                    BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Static | BindingFlags.Instance | BindingFlags.FlattenHierarchy);
+                    BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Static | BindingFlags.Instance);
```

Comment:
It looks like you are removing the two fixes recently submitted for TestCaseSource and TestFixutreSource not finding statics in base classes. Is this intentional? I don't see it mentioned in the PR.